src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -46,11 +51,26 @@ public void Apply(FormulaValue value)
                 _count++;
             }
 
+            public virtual FormulaValue NoElementValue(IRContext context)
+            {
+                return GetDefault(context);
+            }
+
+            public FormulaValue GetDefault(IRContext context)
+            {
+                return new NumberValue(context, _defaultN);
+            }
+
             public virtual FormulaValue GetResult(IRContext irContext)
             {
                 if (_count == 0)
                 {
-                    return new BlankValue(irContext);
+                    return GetDefault(irContext);
```

Comment:
>return GetDefault(irContext); [](http://example.com/codeflow?start=20&length=29)

This is a change from the current behavior; while I think this is correct (`0` is the "neutral / identity" value for sum), it would be good to run through @gregli-msft and @jorisdg before checking this in